### PR TITLE
[net9.0] Merge main into net9.0.

### PR DESCRIPTION
**Pull request by rolfbjarne**
This brings Xcode 16 support to .NET 9.

### CODE REVIEW

File: Make.versions [hunk 1]
```diff
@@ -48,8 +48,8 @@
 #   line changed in git).
 #
 
-IOS_PACKAGE_VERSION=17.5.0.$(IOS_COMMIT_DISTANCE)
-MAC_PACKAGE_VERSION=9.5.0.$(MAC_COMMIT_DISTANCE)
+IOS_PACKAGE_VERSION=17.99.0.$(IOS_COMMIT_DISTANCE)
+MAC_PACKAGE_VERSION=9.99.0.$(MAC_COMMIT_DISTANCE)
```
Comment (dalexsoto) [inline Make.versions:52]: Not that it really matters but… 😃 

```suggestion
IOS_PACKAGE_VERSION=18.0.0.$(IOS_COMMIT_DISTANCE)
MAC_PACKAGE_VERSION=10.0.0.$(MAC_COMMIT_DISTANCE)
```

**Review by dalexsoto — APPROVED** [review]

**Comment by dalexsoto** [comment]
Looks like comp gov got stuck for whatever reason...

File: Make.versions (hunk 1 above)
Comment (rolfbjarne) [reply Make.versions:52]: This will all be removed from main eventually :smile:

**Pull request merged**